workbench-core/accounts/src/services/costCenterService.test.ts

```diff
@@ -115,6 +118,99 @@ describe('CostCenterService', () => {
     });
   });
 
+  describe('list costCenters', () => {
+    const costCenterId = 'cc-someId';
+    const costCenterJson = {
+      pk: `CC#${costCenterId}`,
+      sk: `CC#${costCenterId}`,
+      id: costCenterId,
+      name: 'CostCenter-2',
+      // dependency: 'acc-2272f4a8-1791-419a-8e49-b926f3eb46f9',
+      dependency: accountId,
+      description: 'Description for CostCenter-2',
+      subnetId: accountMetadata.subnetId,
+      vpcId: accountMetadata.vpcId,
+      envMgmtRoleArn: accountMetadata.envMgmtRoleArn,
+      externalId: accountMetadata.externalId,
+      encryptionKeyArn: accountMetadata.encryptionKeyArn,
+      environmentInstanceFiles: accountMetadata.environmentInstanceFiles,
+      hostingAccountHandlerRoleArn: accountMetadata.hostingAccountHandlerRoleArn,
+      awsAccountId: accountMetadata.awsAccountId,
+      createdAt: mockDateObject.toISOString(),
+      updatedAt: mockDateObject.toISOString()
+    };
+    let expectedCostCenter: { [key: string]: unknown } = {
+      ...costCenterJson,
+      accountId: costCenterJson.dependency
+    };
+    expectedCostCenter = removeDynamoDbKeys(expectedCostCenter);
+
+    describe('with more than one "page" of costCenters', () => {
+      test('it return costCenters with a pagination token', async () => {
```

Comment:
maybe "the response returns cost centers and a pagination token"? 'with' sounds like the pagination token is on the cost center objects.